.travis.yml

```diff
@@ -7,28 +7,6 @@ cache:
 
 jobs:
     include:
-        # Run linux as root on linux.
-        # Non-root on linux and root/non_root on OSX are tested via GitHub CI
-        - os: linux
-          python: 2.7
-          env:
-            - TOXENV=py27-linux_root,codecov
-
-        - os: linux
-          python: pypy2
-          env:
-            - TOXENV=pypy-linux_root,codecov
-
-        - os: linux
-          python: pypy3
```

Comment:
On note from my past experience. It might be worth to have one pypy build as root. I often encountered a different behavior of tests, when they run on pypy.

Reply:
I absolutely agree, you'll notice that [2.7, pypy, pypy3, 3.9] all get root builds.
We've had so many bugs with filter compilation on mypy I'm not letting this one slide 😄